src/fn/fn-message-strip.scss

```diff
@@ -0,0 +1,101 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-message-strip;
+
+.#{$block} {
+  @include fn-reset();
+  @include fn-flex();
+  @include fn-set-paddings-x(0.5rem, 0.5rem);
+
+  align-items: center;
+  background-color: $fn-color-blue-50;
+  color: $fn-color-grey-700;
+  min-height: 1.75rem;
+  min-width: 20.5rem;
+  width: fit-content;
+  border-radius: $fn-border-radius-4;
+
+  &__text {
+    @include fn-reset();
+
+    color: inherit;
+    max-width: 95%;
+  }
+
+  &__icon {
+    background-color: transparent;
+    margin-right: 0.25rem !important;
+
+    @include fn-rtl() {
+      margin-right: 0 !important;
+      margin-left: 0.25rem !important;
+    }
+
+    &::before {
+      color: $fn-color-blue-500;
+      display: flex;
+      margin-top: 0.125rem;
+    }
+  }
+
+  &--dismissible {
```

Comment:
Why do you need a modifier class? If it's a dismissible message strip we can just add button to the markup.